go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -91,15 +91,15 @@ Otherwise, the keyspace must be empty (have no shards), or returns an error.`,
 	// SetKeyspaceServedFrom makes a SetKeyspaceServedFrom gRPC call to a vtcltd.
 	SetKeyspaceServedFrom = &cobra.Command{
 		Use:                   "SetKeyspaceServedFrom [--source <keyspace>] [--remove] [--cells=<cells>] <keyspace> <tablet_type>",
-		Short:                 "Updates the ServedFromMap for a keyspace manually. This command is intended for emergency fixes; the map is automatically set by MigrateServedTypes. This command does not rebuild the serving graph.",
+		Short:                 "(DEPRECATED) Updates the ServedFromMap for a keyspace manually. This command is intended for emergency fixes. This command does not rebuild the serving graph.",
```

Comment:
You should also set the [`Deprecated`](https://pkg.go.dev/github.com/spf13/cobra#Command) field on these commands, which "defines, if this command is deprecated and should print this string when used."

Reply:
Thanks! I didn't notice that it also existed here. Will change now.